### PR TITLE
Check for tsconfig only if any of the TS function

### PR DESCRIPTION
**Pull request by veljko-matic**

### CODE REVIEW

File: commands/actions/publish.go [hunk 1]
```diff
@@ -95,13 +96,10 @@ func buildFunc(cmd *cobra.Command, args []string) {
...
 
-	existsTsFiles, err := util.TsFilesExists(actions.Sources)
-	if err != nil {
-		logrus.Warn("\nError while detecting .ts files in sources")
-	}
 	tsConfigExists := util.TsConfigExists(actions.Sources)
 
-	if existsTsFiles && !tsConfigExists {
+	if anyFunctionTsFileExists(actions) && !tsConfigExists {
+		err := errors.New("Some of the functions are with .ts extension but there is no typescript config file!")
```
Comment (BogdanHabic) [inline commands/actions/publish.go:102]: I would return the concrete file that returned the error so the user doesn't have to hunt around for it.

File: commands/actions/publish.go [hunk 2]
```diff
@@ -356,3 +354,27 @@ func mustValidateTsconfig(tsconfig *typescript.TsConfig) {
 		os.Exit(1)
 	}
 }
+
+func anyFunctionTsFileExists(actions *actionsModel.ProjectActions) bool {
+	for _, spec := range actions.Specs {
+		locator := spec.Function
+		internalLocator, err := actionsModel.NewInternalLocator(locator)
+		if err != nil {
```
Comment (BogdanHabic) [inline commands/actions/publish.go:362]: Does this check if the file/function exists?

**Review by BogdanHabic — APPROVED** [review]
One question and change and LGTM. No need for another review

File: commands/actions/publish.go (hunk 2 above)
Comment (veljko-matic) [reply commands/actions/publish.go:362]: No it doesn't. This only check ->  is there two parts when we split string with `:` separator. But we have `mustGetSource` that check file existence.

**Pull request merged**